runtime/runtime-params-estimator/Cargo.toml

```diff
@@ -69,9 +69,13 @@ nightly_protocol = [
   "near-primitives/nightly_protocol",
   "near-test-contracts/nightly",
   "protocol_feature_nep366_delegate_action",
+  "protocol_feature_flat_state",
```

Comment:
I believe this should be a part of `nightly` to be consistent with `nearcore/Cargo.toml`

Reply:
Hm, makes sense. (Although, I must admit I'm never sure what the difference between "nightly" and "nightly_protocol" is anyway.)